packages/sdk/src/Portfolio/Integrations/MysoV3.ts

```diff
@@ -0,0 +1,396 @@
+}
+
+export function closeAndSweepEscrowsDecode(encoded: Hex): CloseAndSweepEscrowsArgs {
+  const [{ escrowIdxs, skipWithdrawFromEscrow }] = decodeAbiParameters(closeAndSweepEscrowsEncoding, encoded);
+  return { escrowIdxs, skipWithdrawFromEscrow };
+}
+
+//------------------------------------------------------------------------------
+// WITHDRAW TOKENS FROM ESCROWS
+//------------------------------------------------------------------------------
+
+const withdrawTokensFromEscrowsEncoding = [
+  {
+    type: "tuple",
+    name: "withdrawTokensFromEscrowsArgs",
+    components: [
+      {
+        type: "address[]",
+        name: "escrows",
+      },
+      {
+        type: "address[]",
+        name: "tokens",
+      },
+    ],
+  },
+] as const;
+
+export function withdrawTokensFromEscrowsEncode(args: WithdrawTokensFromEscrowsArgs): Hex {
+  return encodeAbiParameters(withdrawTokensFromEscrowsEncoding, [args]);
+}
+
+export function withdrawTokensFromEscrowsDecode(encoded: Hex): WithdrawTokensFromEscrowsArgs {
+  const [{ escrows, tokens }] = decodeAbiParameters(withdrawTokensFromEscrowsEncoding, encoded);
+  return { escrows, tokens };
+}
+
+//------------------------------------------------------------------------------
+// SWEEP
+//------------------------------------------------------------------------------
+
+const sweepEncoding = [
+  {
+    type: "tuple",
+    name: "sweepArgs",
+    components: [
+      {
+        type: "address[]",
+        name: "tokens",
+      },
+    ],
+  },
+] as const;
+
+export function sweepEncode(args: SweepArgs): Hex {
+  return encodeAbiParameters(sweepEncoding, [args]);
+}
+
+export function sweepDecode(encoded: Hex): SweepArgs {
+  const [{ tokens }] = decodeAbiParameters(sweepEncoding, encoded);
+  return { tokens } as const;
+}
+
+//------------------------------------------------------------------------------
+// ACTION FUNCTIONS
+//------------------------------------------------------------------------------
+export const createEscrowByTakingQuote = ExternalPositionManager.makeUse(
+  Action.CreateEscrowByTakingQuote,
+  createEscrowByTakingQuoteEncode,
+);
+
+export const createEscrowByStartingAuction = ExternalPositionManager.makeUse(
+  Action.CreateEscrowByStartingAuction,
+  createEscrowByStartingAuctionEncode,
+);
+
+export const closeAndSweepEscrows = ExternalPositionManager.makeUse(
+  Action.CloseAndSweepEscrows,
+  closeAndSweepEscrowsEncode,
+);
+
+export const withdrawTokensFromEscrows = ExternalPositionManager.makeUse(
+  Action.WithdrawTokensFromEscrows,
+  withdrawTokensFromEscrowsEncode,
+);
+
+export const sweep = ExternalPositionManager.makeUse(Action.Sweep, sweepEncode);
+
+//------------------------------------------------------------------------------
+// EXTERNAL READ FUNCTIONS
+//------------------------------------------------------------------------------
+export function getManagedAssets(client: Client, args: Viem.ContractCallParameters<{ position: Address }>) {
```

Comment:
it is already in the sdk under `getExternalPositionManagedAssets`

Reply:
removed!